translations/sv/learn.json

```
@@ -0,0 +1,31 @@
+{
+  "diving_into_fediverse": "Dyka ner i <mark>fediversumet</mark>",
+  "what_is_fediverse": "<p><a href='https://en.wikipedia.org/wiki/Fediverse'>Fediversumet</a> är <mark>enormt</mark>. Det består av tusentals gemenskaper med miljontals människor som du och jag, som vill <mark>ansluta och dela</mark>.</p><p>Här är några resurser som hjälper dig att få en bättre uppfattning om hur saker fungerar, hur du hittar en gemenskap som passar dig och utforskar verktyg du kan använda för att göra din upplevelse ännu bättre.</p>",
+  "faq_title": "Vanliga frågor",
+  "faq_instance_q": "F: Vad är en \"instans\" eller \"server\"? Hur väljer jag en?",
+  "faq_instance_a": "<p><strong>S:</strong> Fediversumet består av distinkta gemenskaper, även kallade <mark>\"instanser\"</mark> eller <mark>\"servrar\"</mark>. Du kan till och med <a href='https://stefanbohacek.com/blog/lessons-learned-from-running-a-single-person-mastodon-instance/#hosting'>driva din egen gemenskap</a>. Och det bästa är att oavsett vilken server du registrerar dig på, kan du alltid nå vem som helst i hela fediversumet.</p><p>Att flytta mellan olika gemenskaper är relativt enkelt. Du kan <mark>ta med dina vänner och följare</mark>, så oroa dig inte för att välja rätt instans från början. Registrera dig bara, känn efter, och du kan flytta till en annan server om du hittar något som passar bättre.</p>",
```

```suggestion
  "faq_instance_a": "<p><strong>S:</strong> Fediversumet består av distinkta gemenskaper, även kallade <mark>\"instanser\"</mark> eller <mark>\"servrar\"</mark>. Du kan till och med <a href='https://stefanbohacek.com/blog/lessons-learned-from-running-a-single-person-mastodon-instance/#hosting'>driva din egen gemenskap</a>. Och det bästa är att oavsett vilken server du registrerar dig på, kan du alltid nå vem som helst i hela fediversumet.</p><p>Att flytta mellan olika gemenskaper är relativt enkelt. Du kan <mark>ta med dina vänner och följare</mark>, så oroa dig inte för att välja rätt instans från början. Registrera dig bara, känn efter, och flytta till en annan server om du hittar något som passar bättre.</p>",
```
Make text flow better